### ISSUE TITLE
Scrollbar should not change size whilst being dragged

### ISSUE THREAD

**Issue by Bilge** [utatti/perfect-scrollbar#736]
As seen in the [official demo](http://utatti.github.io/perfect-scrollbar/), the scrollbar becomes bigger and easier to click as the mouse moves near it, and shrinks again as the mouse moves away. However, whilst being clicked and dragged this effect continues, whilst I opine it should not: the scrollbar should only be allowed to shrink again once the mouse is released and the dragging ends.

When the scrollbar shrinks during dragging, it makes the scroll position harder to see and seems to imply the drag event has ended prematurely (even though it continues to work).

---
- [x] Check [FAQ](https://github.com/utatti/perfect-scrollbar/wiki/FAQ) and [Caveats](https://github.com/utatti/perfect-scrollbar/wiki/Caveats)
- [x] Search if there's already one reported in Issues
- [ ] Prepare a JSFiddle reproducing the issue
  - [perfect-scrollbar JSFiddle](https://jsfiddle.net/utatti/dyvL31r6/)
- [x] Provide a page or source code where the issue can be checked

**Comment by utatti** [comment]
I think it's a valid point. Will do this.

**Comment by utatti** [comment]
It is how scrollbars show on macOS and I personally think it's okay, but I agree that it would be nice to have an option to style. I added `ps--clicking-x` and `ps--clicking-y` classes on dragging and you can now easily customize the case, with the following CSS.

```css
.ps--scrolling-y .ps__rail-y {
  background-color: #eee;
  opacity: 0.9;
}

.ps--scrolling-y .ps__thumb-y {
  background-color: #999;
  width: 11px;
}
```

**Comment by Bilge** [comment]
At first you agreed with me but now you changed your opinion because Mac does it differently. It's not as if Mac is some holy design epitome of unfathomable perfection. This should not be the default behaviour no matter what Apple does.

**Comment by utatti** [comment]
I did delete the previous comment, not because I changed my opinion but because I misunderstood the behaviour you described.

I do not (and never) argue that *Mac is some holy design epitome of unfathomable perfection*. The CSS perfect-scrollbar provides is rather an example or bootstrap, not *default behaviour*, and I wish users change it as they want. I just decided to choose macOS scrollbar behaviour for the example, because I'm a macOS user and do not own any copy of other OS.

I'm sorry again that the behaviour does not satisfy your point of view.
